applications/RomApplication/rom_application_variables.cpp

```diff
@@ -0,0 +1,21 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:        BSD License
+//                  Kratos default license: kratos/license.txt
+//
+//  Main authors:   Raul Bravo
+//
+//
+
+
+#include "rom_application_variables.h"
+
+namespace Kratos
+{
+    KRATOS_CREATE_VARIABLE( int, AUX_ID )

```

Comment:
I suggest to find a more descriptive name.

Reply:
This variable is only used as an auxiliary ID when the total number of elements and the ones used for assembly do not match (hyper-reduction).

Reply:
Then I'd call it `HR_AUX_ID`.